src/API/users.api.json

```diff
@@ -0,0 +1,21 @@
+{
+    "routes":{
+        "getUsers": {
+            "http": {
+                "method": "GET",
+                "request_uri": "/users"
+            },
+            "params": {
+                "page": {
+                    "optional": false
+                },
+                "page_size": {
+                    "optional": false
+                }
+            },
+            "output": {
+                "array":"\\Ionic\\Users\\Models\\User"
```

Comment:
The output of the collection endpoints sometimes contains meta information, such as the total number of users (if requested). It may be better to wrap these in some sort of Collection object?